### PR TITLE
Add product id’s from CMP GraphQL query to response headers

### PR DESCRIPTION
**Pull request by supercid**

### CODE REVIEW

File: Helper/ProductDebugHelper.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+ *
+ * @author Nosto Solutions Ltd <contact@nosto.com>
+ * @copyright 2019 Nosto Solutions Ltd
+ * @license http://opensource.org/licenses/BSD-3-Clause BSD 3-Clause
+ *
+ */
+
+namespace Nosto\Cmp\Helper;
+
+class ProductDebugHelper
```
Comment (mridang) [inline Helper/ProductDebugHelper.php]: Remove the word Helper. The class needs a new name

File: Helper/ProductDebugHelper.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    /**
+     * @return string
+     */
+    public function build()
+    {
+        $value = '';
+        foreach ($this->productIds as $name => $productId) {
+            $productIdsString = '';
+            foreach ($productId as $item) {
```
Comment (mridang) [inline Helper/ProductDebugHelper.php]: Why not `implode `?

File: Helper/ProductDebugHelper.php [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    private function __construct()
+    {
+        // Private
+    }
+
+    /**
+     * @param array $ids
+     * @param $name
+     */
+    public function addProductIds(array $ids, $name)
```
Comment (mridang) [inline Helper/ProductDebugHelper.php]: What is the name parameter for?

Comment (supercid) [reply Helper/ProductDebugHelper.php]: True, unecessary

**Pull request merged**